src/core/bitop.d

```diff
@@ -72,20 +72,17 @@ unittest
  */
 int bsf(size_t v) pure;
 
+version(CoreDdoc)
 /// ditto
+int bsf(ulong v) pure;
+
+version(D_LP64) { } else
```

Comment:
> it is documenting the fact that this code depends upon the precise bit-ness of the types involved

And this is what `D_LP64` for. If someone don't know about `D_LP64` it's not our problem. It's silly to not use language constructs just because someone can be uninformed about it.

> But, trying to change the style of a module by updating some random small piece of it just increases the chaos.

Yes, but I will not use the style I'm against of.